source/guides/faq.md

```diff
@@ -49,6 +49,44 @@ See <https://github.com/nix-community/home-manager>
 
 ## NixOS
 
+### How to run non-nix executables?
+
+NixOS cannot run dynamically linked executables intended for generic linux environments out of the box.
+This is because, by design, it does not have a global library path.
+Solutions include:
+
+- Use the version packaged in nixpkgs, if there is one.
+  You can search available packages at <https://search.nixos.org/packages>.
+
+- Write a nix expression for the program to package it in your own configuration.
+  There are multiple approaches:
+  - Build from source.
+    For an introduction to this, see [](../tutorials/packaging-existing-software).
+  - Set the program's ELF metadata to include paths to libraries using [`autoPatchelfHook`](https://nixos.org/manual/nixpkgs/stable/#setup-hook-autopatchelfhook).
+    Do this if building from source isn't feasible.
+  - Wrap the program to run in a normal-seeming container using [`buildFHSEnv`](https://nixos.org/manual/nixpkgs/stable/#sec-fhs-environments).
```

Comment:
I was trying to avoid the word "FHS" in this message. It's jargon that's basically only known in the nixos community, because we need word for what we aren't. The rest of the world doesn't need a word for it. And this message is likely to be many reader's introduction to the fact that nixos is different in this way.